plugins/polio/api/vaccines/supply_chain.py

```diff
@@ -301,10 +310,25 @@ class Meta:
             "comment",
             "target_population",
             "vrf_type",
+            "document",
         ]
 
         read_only_fields = ["created_at", "updated_at"]
 
+    def to_internal_value(self, data):
+        # Manually invoke validate_rounds if 'rounds' is a string
+        if "rounds" in data and isinstance(data["rounds"], str):
+            try:
+                rounds = parse_comma_separated_numeric_values(data["rounds"], "rounds")
+                data["rounds"] = [{"number": num} for num in rounds]
+            except Exception as e:
+                raise serializers.ValidationError(f"Invalid rounds data: {e}")
+        return super().to_internal_value(data)
```

Comment:
It's only related because we changed from JSON to FormData to send the data to the server @bramj and so the old way of sending the rounds was not working